src/core/core.js

```diff
@@ -140,6 +157,10 @@ class Core {
     return this;
   }
 
+  get interceptors () {
```

Comment:
My only comment here is why do we need a getter for this?

Reply:
Just put this there to be access them easier e.g.:
```javascript
boot(){
    this.interceptors.request.push(...);
}
```